saleor/search/search_indexes.py

```diff
@@ -0,0 +1,22 @@
+from haystack import indexes
+from ..product.models import ProductVariant, Product
+
+
+class ProductVariantIndex(indexes.SearchIndex, indexes.Indexable):
+    text = indexes.CharField(document=True, use_template=True)
```

Comment:
Can we please not use a single document field for search? It does not what which field is which so it'll treat the product name, variant name and description as a single value. None of the metrics like term/field factor or term position make any sense in that setup.